lib/components/table.vue

```diff
@@ -76,14 +76,15 @@
 </template>
 
 <script>
-    import {warn} from '../utils';
+    import { warn } from '../utils';
+    import { keys } from '../utils/object';
```

Comment:
interesting, if we can do this like  `import { object.keys  as keys } from '../utils';`